app/features/security-txt/ui/SecurityTXTLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import { PublicKey } from '@solana/web3.js';
+import Link from 'next/link';
+import React from 'react';
+import { ExternalLink } from 'react-feather';
+
+import { InfoTooltip } from '@/app/components/common/InfoTooltip';
+import { useProgramMetadataSecurityTxt } from '@/app/entities/program-metadata';
+import { useCluster } from '@/app/providers/cluster';
+
+import { NEODYME_SECURITY_TXT_DOC_LINK, PMP_SECURITY_TXT_DOC_LINK } from '../lib/constants';
+
+export function ProgramSecurityTXTLabel({ programPubkey }: { programPubkey: PublicKey }) {
+    const { url, cluster } = useCluster();
+    const { programMetadataSecurityTxt } = useProgramMetadataSecurityTxt(programPubkey.toBase58(), url, cluster);
+
+    return (
+        <InfoTooltip text="Security.txt helps security researchers to contact developers if they find security bugs.">
+            {/* Reference by default to Neodyme security.txt doc */}
+            {/* Reference to Program Metadata only if it's uploaded */}
```

Comment:
Could we maybe add a link to how to upload PMP Security.txt if there is none uploaded?

Reply:
What do you think about adding a copyable line of script with that command? Would it be helpful in your opinion?

Reply:
Which command do you mean? 
I think just a text and link to https://github.com/solana-program/program-metadata would be enough.

Reply:
<img width="702" height="611" alt="image" src="https://github.com/user-attachments/assets/25b55a27-8817-45f3-a7d1-5b7033fed7a7" />
There is an info message in purple (that is a destructive color for the explorer) that states the data is self-reported. I think it covers the disclaimer. WDYT?